core/contracts/oracle/interfaces/VotingInterface.sol

```diff
@@ -14,6 +13,9 @@ contract VotingInterface {
         uint time;
     }
 
+    // Note: the phases must be in order. Meaning the first enum value must be the first phase, etc.
+    enum Phase { Commit, Reveal, PLACEHOLDER_LAST_VALUE }
```

Comment:
@chrismaree @nicholaspai any suggestions for more readable names for a placeholder value that's used to compute the length of the enum?

Reply:
Why not name this NUM_PHASES, the value its replacing

But this value doesn't make sense to store in the `Phase` enum, why not just make it a public variable?

Reply:
Because we can't access public variables from this interface in the library where we want to use it. We can only access *types*.

Reply:
Were you able to come up with other ideas? I'd be fine with NUM_PHASES_PLACEHOLDER.

I think it'd be helpful to put something in the enum name to make it clear that this value is special and should not be set.

Reply:
NUM_PHASES_PLACEHOLDER SGTM. I would also add a comment that this value must always remain at the end.

Reply:
Done.